content/docs/user-guide/building/venv.md

```diff
@@ -0,0 +1,50 @@
+# Virtual Environments
+
+Create a python virtual environment using requirements gathered from a model.
```

Comment:
same, I would clarify the use case

Reply:
I'd also consider moving it to the section index page bc/ this doesn't actually export/build the model right? It seems like another utility around building.

Reply:
@jorgeorpinel, do you propose to have a links to all nested pages in the index page? Something like we did [here](https://mlem-ai-new-docs-struct-hy95mv.herokuapp.com/doc/get-started#productionization). So somewhere on [this page ](https://mlem-ai-new-docs-struct-hy95mv.herokuapp.com/doc/user-guide/building) we need to have links + short descriptions to the child pages? Sounds like a good idea to me!

Reply:
Ah, looks like I misinterpreted what you said here. Since you left the same comment below, I've answered to your point there. Still, do you think it worth writing these short lists in the end of each page? On the second though, this looks like duplicating effort to me, so I think we should skip it for now.

Reply:
> have a links to all nested pages in the index page?

Not exactly. I meant [this](https://github.com/iterative/mlem.ai/pull/191#discussion_r1000027203).

> do you think it worth writing these short lists in the end of each page?

Not sure. I mean linking between pages in general is great of course, but making an explicit list that repeats the left side nav seems unnecessary (possible maintenance headache too). In dvc.org/doc we tend to mention the concepts and link them that way (more casually). E.g. see https://dvc.org/doc/user-guide/project-structure/dvcyaml-files

Reply:
I added a use-case, not sure what the conclusion of the other discussion pertaining to `index` stuff is.